Automattic/jetpack
# Debugger: Integration for WordPress 5.2 Site Health functionality

WordPress 5.2 adds in a site health "testing suite" and common debug data source. This PR would add a single test to check for the overall local testing suite status and moves the Jetpack Advanced Debug Data to a section in the new debug data area.

#### Changes proposed in this Pull Request:
* WordPress 5.2 introduces a new Site Health tool and Jetpack integrates wonderfully with it!

#### Testing instructions:
* Connected Jetpack site running WP 5.2 beta.
* On 5.2-beta, visit Tools->Site Health. Does the Jetpack test run async (e.g. delayed response to allow time for remote http calls.
* Visit Debug Data. Does Jetpack have a section there?
* Press the copy to clipboard button. Paste. Are the user/blog tokens NOT present.
* On WP 5.0 or 5.1, anything weird? Visit the Jetpack Debugger -- verify the debug is still there (albeit with different more human readable labels)

#### Proposed changelog entry for your changes:


*
